xbmc/Application.cpp

```diff
@@ -2939,6 +2939,36 @@ void CApplication::Stop(int exitCode)
   Sleep(200);
 }
 
+bool CApplication::ResumePlayer(const CFileItemPtr& itemIn) {
+	CLog::Log(LOGDEBUG, "CApplication::ResumePlayer resume item %s", itemIn->GetPath().c_str());
+	if (m_pPlayer->HasPlayer())
+	{
+		CLog::Log(LOGWARNING, "CApplication::ResumePlayer can't resume, player is already startet");
+		return false;
+	}
+
+	CFileItem item(itemIn->GetItemToPlay());
```

Comment:
I don't know whis this was part of the resume function. Should we keep it or think about it?

Reply:
It's only needed for files which are not loaded in the player. I would keep if if we want to call this funktion from by contextMenu